### ISSUE TITLE
EPoch number does not make sense

### ISSUE THREAD

**Issue by remybonnav** [Linaqruf/kohya-trainer#82]
I am training a dreambooth LORA,

Í have 18 training image, 1 repeat, batch = , gradient accumulation = 6, and 200 epoch, gradient checkpointing true.
But the output cell when starting training indicates that the training will last 1200 epochs.

It should be 200, not 1200 right?

**Comment by Linaqruf** [comment]
gradient accumulation steps is basically batch size but for memory efficiency. 

```python
  # epoch数を計算する
  num_update_steps_per_epoch = math.ceil(len(train_dataloader) / args.gradient_accumulation_steps)
  num_train_epochs = math.ceil(args.max_train_steps / num_update_steps_per_epoch)
  if (args.save_n_epoch_ratio is not None) and (args.save_n_epoch_ratio > 0):
    args.save_every_n_epochs = math.floor(num_train_epochs / args.save_n_epoch_ratio) or 1
```

so to get valid max train steps, you need to multiply max_train_steps with args.gradient_accumulation_steps, 

`200*6 = 1200 steps.`

**Comment by remybonnav** [comment]
Ok thanks.
